data_model/src/query/mod.rs

```diff
@@ -970,18 +962,39 @@ pub mod peer {
         #[display(fmt = "Find all peers")]
         #[ffi_type]
         pub struct FindAllPeers;
+    }
+
+    /// The prelude re-exports most commonly used traits, structs and macros from this crate.
+    pub mod prelude {
+        pub use super::FindAllPeers;
+    }
+}
+
+pub mod executor {
+    //! Queries related to [`crate::executor`].
+
+    #[cfg(not(feature = "std"))]
+    use alloc::{format, string::String, vec::Vec};
+
+    use derive_more::Display;
+
+    queries! {
+        /// [`FindExecutorDataModel`] Iroha Query finds the data model of the current executor.
+        #[derive(Copy, Display)]
+        #[display(fmt = "Find executor data model")]
+        #[ffi_type]
+        pub struct FindExecutorDataModel;
 
-        /// [`FindAllParameters`] Iroha Query finds all [`Peer`]s parameters.
+        /// [`FindAllParameters`] Iroha Query finds all defined executor configuration parameters.
         #[derive(Copy, Display)]
         #[display(fmt = "Find all peers parameters")]
-        // TODO: Unused query. Remove?
         #[ffi_type]
         pub struct FindAllParameters;
```

Comment:
I feel we have a small discrepancy here. On the one hand we have `FindAllParameters` but we don't have `FindAllPermissions` (I assume this would be for some account)?

Reply:
> (I assume this would be for some account)

Yes, currently there is `FindPermissionsByAccountId`, and that is the only permission-related query. There are also roles-related though.